src/config/dynamic-theme-fixes.config

```diff
@@ -1080,6 +1080,12 @@ INVERT
 .DatePickerCalendarDate--today .DatePickerCalendarDate-button::after
 
 CSS
+.MultiColorIcon-path--fadedBlack {
+    fill: ${#AAA} !important;
+}
+.MultiColorIcon--unselected .MultiColorIcon-path--fadedWhite {
+    fill: ${#FFF} !important;
```

Comment:
```suggestion
    fill: var(--darkreader-neutral-background) !important;
```

Reply:
I tried that, but it didn't help, so I did something completely different, and that works. I'll put that change in a new commit.